lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,70 @@
+namespace :data do
+  desc 'Create demo data for our local development'
+  task test: :environment do
+    include FactoryGirl::Syntax::Methods
+
+    def generate_events program, conference
+        user1 = create(:user)
+        user2 = create(:user)
+
+        selected_schedule = create(:schedule, program: program)
+        demo_schedule = create(:schedule, program: program)
+        program.update_attributes(selected_schedule: selected_schedule)
+
+        build(:event, program: program, title: 'Demo Event', abstract: 'This is a demo event instance whose state not defined.').save(validate: false)
```

Comment:
> You can not introduce non valid records in our database because we are assuming everywhere that the records are valid, they should be valid.

But we do have some conferences in our db that has already happened. I don't think these records will cause any such trouble though.